src/components/Card/Card.js

```diff
@@ -1,11 +1,26 @@
 import './Card.css';
+import { processText } from '../../utils/helpers';
 
 function Card(props) {
```

Comment:
I think using destructor is more readable and safe to support  instead of  props.
```
function({card, type}){
...
}
```

Reply:
I think that's debatable, actually, but I don't mind changing it.